lnd_test.go

```diff
@@ -67,6 +68,8 @@ func newHarnessTest(t *testing.T) *harnessTest {
 // integration tests should mark test failures solely with this method due to
 // the error stack traces it produces.
 func (h *harnessTest) Fatalf(format string, a ...interface{}) {
+	lndHarness.SaveProfilesPages()
+
```

Comment:
I think you can use lntest.NetworkHarness.SaveProfilesPages()

Reply:
If we run it with nil pointer we won't get anything. We must point to NetworkHarness that we initialize in TestLightningNetworkDaemon.
Or we can declare *lntest.NetworkHarness in TestLightningNetworkDaemon() as before and use SaveProfilesPages() before all Fatalf().